driver-scala/src/main/scala/org/mongodb/scala/ObservableImplicits.scala

```diff
@@ -174,9 +174,9 @@ trait ObservableImplicits {
     val observable = obs
 
     /**
-     * Collects the [[Observable]] results and converts to a [[scala.concurrent.Future]].
+     * Collects the [[Observable]] results and converts to a scala.concurrent.Future`.
      *
-     * Automatically subscribes to the `Observable` and uses the [[collect]] method to aggregate the results.
+     * Automatically subscribes to the `Observable` and uses the [[Observable.collect]] method to aggregate the results.
```

Comment:
When do you use brackets and when to do you use backticks?

Reply:
Brackets are for links, backticks are for monospace text.